daphne_worker/src/config.rs

```diff
@@ -837,6 +837,7 @@ impl<'srv> DaphneWorker<'srv> {
                 let bits = bits.parse().map_err(int_err)?;
                 VdafConfig::Prio3(Prio3Config::Sum { bits })
             }
+            // TODO(tholop): should we add SumVec too? Why is Histogram not here?
```

Comment:
Yes, please add it! Histogram should be here, too, but we'll add it in a PR for https://github.com/cloudflare/daphne/issues/271.

Reply:
I added SumVec there, but it required me to extend `InternalTestVdaf`. I don't know how to run the interop tests, so not sure if it breaks anything

Reply:
We don't have interop tests running at the moment. If you'd like to add tests, we could add unit tests for decoding a JSON blob with the config we're expecting.

Reply:
I added a new issue for that (https://github.com/cloudflare/daphne/issues/333), it might overlap with https://github.com/cloudflare/daphne/issues/271.